python/migrate.py

```diff
@@ -115,81 +255,48 @@ def oracle_db(
         else:
             cursor.execute(table_or_sql, **params)
 
-        return [
-            mgp.Record(row=_name_row_cells(raw_row, cursor))
-            for raw_row in cursor.fetchall()
+        oracle_db_dict[threading.get_native_id][Constants.CONNECTION] = connection
+        oracle_db_dict[threading.get_native_id][Constants.CURSOR] = cursor
+        oracle_db_dict[threading.get_native_id][Constants.COLUMN_NAMES] = [
+            column[Constants.I_COLUMN_NAME] for column in cursor.description
         ]
 
 
-@mgp.read_proc
-def sql_server(
-    config: Any,
+def oracle_db(
```

Comment:
All parameters unused

Reply:
Yeah that's by design of batched procedures.